modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -3,12 +3,32 @@
 using namespace cv;
 
 unsigned countCoins(const Mat& img) {
-    Mat gray, thresh;
-    cvtColor(img, gray, COLOR_BGR2GRAY);
-    threshold(gray, thresh, 0, 255, THRESH_BINARY_INV + THRESH_OTSU);
-    // Remove unfilled black holes by morphological closing
-    morphologyEx(thresh, thresh, MORPH_CLOSE, Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
-
-    // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
-    CV_Error(Error::StsNotImplemented, "countCoins");
+	Mat gray, thresh;
```

Comment:
Please remove redundant spaces, yours differences with old version not right

Reply:
In visual studio correctly displays, but here for some reason they are shifted

Reply:
please correct this

Reply:
You can find a settings like "Replace Tab to Spaces" or "Number of whitespaces in Tab"